irods/path/__init__.py

```diff
@@ -9,20 +9,39 @@ class iRODSPath(str):
 
     def __new__(cls, *elem_list, **kw):
         """
-        Initializes our immutable 'str' object with a normalized form.
+        Initializes our immutable string object with a normalized form.
+        An instance of iRODSPath is also a `str'
 
-        Keywords may include 'absolute'. The default is True, forcing a slash as
-        the leading character of the resulting string object.
+        Keywords may include only 'absolute'. The default is True, forcing a slash as
+        the leading character of the resulting string.
 
         Variadic parameters are the path elements, strings which may name individual
-        collections or sub-hierarchies (internally slash-separated):
+        collections or sub-hierarchies (internally slash-separated).  These are then
+        joined using the path separator:
 
-        In the resulting object, trailing and redundant path separator are removed,
-        as is the "trivial" path element ('.'):
+          data_path = iRODSPath( 'myZone', 'home/user', './dir', 'mydata')
+          # => '/myZone/home/user/dir/mydata'
```

Comment:
yeps.

indent a couple more spaces to match the ones below...

Reply:
done.